src/fcp-decisions.md

```diff
@@ -0,0 +1,76 @@
+FCP decisions represent a significant commitment from the team. They require sign-off from team members and follow a formal process for resolving concerns. An FCP decision cannot be reversed without another FCP—the expectation is that reversal requires some change in circumstances: new experience, new information, or a reasoned change of mind.
+
+FCP decisions are used for stabilization, RFC approval, and other cases where we are making a promise to our users or taking a position we don't want to reverse lightly.
+
+## Definitions
+
+* **Lang team**: The [Language Design Team](https://github.com/rust-lang/team/blob/master/teams/lang.toml).
+* **Lang team member**: A member of the lang team. Lang team members are the ones who approve the final decision.
+* **Advisor**: A member of the [lang team advisors](https://github.com/rust-lang/team/blob/master/teams/lang-advisors.toml). Advisors can propose a decision and can raise concerns, but their approval is not required.
+* **Decision document**: The RFC, issue text, or other document describing the decision being made.
+* **Document author**: The person who authored the decision document. They ultimately decide what changes they wish to make in response to concerns.
+
+## The process
+
+FCP decisions use rfcbot. They always begin with a decision document authored by the document author, who can be anyone—they do not have to be a member of a Rust team.
+
+To start an FCP, a lang team member or advisor issues `@rfcbot fcp merge` (or `fcp close` for closing). This creates checkboxes. Once enough boxes have been checked (per rfcbot's standard rules), the decision enters final-comment-period. Assuming no concerns are raised, the decision is finalized once the FCP has expired.
+
+Before finalizing the decision, the team should engage with any **new** points raised during the FCP period, particularly from people not able to raise formal concerns.
+
+## Raising concerns
+
+Lang team members or advisors can raise a **formal concern** during the discussion process using `@rfcbot concern concern-name`. The decision cannot be finalized until the concern is resolved.
+
+### Expectations when raising a concern
+
+The person who raises a concern is expected to:
+
+* Write a constructive comment explaining the concern
+* Make themselves available for discussion in a reasonable fashion
+* Be prepared to write a summary document if requested
+
+## Resolving concerns
+
+There are two ways to resolve a concern:
+
+### 1. Withdrawn by the person who raised it
+
+If the person who raised the concern is satisfied—whether because of changes made or because they've decided not to block progress—they resolve the concern themselves with `@rfcbot resolve`. This is the preferred outcome.
+
+### 2. Resolution proposed
+
+If the concern is not withdrawn, someone (e.g. the document author) can propose a **resolution**. This is a more formal process that ensures concerns are genuinely engaged with.
+
+#### Creating a concern issue
```

Comment:
I think this should go under the "raising concerns" section rather than "resolving concerns". Otherwise it implies that the only reason to open an issue if you want to file a formal resolution, which I don't think is the intention.